provisionersdk/scripts/bootstrap_linux.sh

```diff
@@ -86,4 +86,12 @@ fi
 
 export CODER_AGENT_AUTH="${AUTH_TYPE}"
 export CODER_AGENT_URL="${ACCESS_URL}"
-exec ./$BINARY_NAME agent
+
+output=$(./${BINARY_NAME} --version | head -n1)
+if ! echo "${output}" | grep -q Coder; then
+	echo >&2 "ERROR: Downloaded agent binary is invalid"
+	echo >&2 "Script output: '${output}'"
```

Comment:
single quotes?

Reply:
IIRC I ran into some trouble with double-quotes, lemme refresh my memory or fix.

Reply:
There's an unwritten contract here that this must work correctly with quoting in Terraform and when passed to `sh -c $input`, it's a bit dicey and easy to break.

Reply:
Ah yes, good ol' Powershell. I wanted a unified format across all 3 platforms. I've figured out how to double-quote now though: see 70e3091ced28a8af2bfba5315b7d759cbc136d86